### DISCUSSION TITLE
Check out Cormorant from Eclectic Light Co/Howard Oakley

### DISCUSSION THREAD

**Discussion by hstriepe** [aonez/Keka#1106] [Ideas]
Howard Oakley at eclecticlight.co did some experimentation with compression on Apple Silicon in his cormorant utility.

In the fast mode using all cores and based on Apple's framework this thing is unbelievable fast! Might be good to have a look.

**Comment by gingerbeardman** [comment]
Such features have already been integrated in the latest Keka beta. See https://github.com/aonez/Keka/issues/1087

**Comment by aonez** [comment]
As @gingerbeardman that build in #1087 has additional QoS selection. Also since version 1.2.50 you can compress and extract Apple Archive format support.
